### DISCUSSION TITLE
Cannot search anything on a specific repo

### DISCUSSION THREAD

**Discussion by willin** [community/community#15191] [Projects and Issues]
https://github.com/Authing/docs

search anything, found nothing... such as:

https://github.com/Authing/docs/search?q=express

**Comment by FilipKufalov** [comment]
If you trying to find specific files according to the given filename you can use the next method

![image](https://user-images.githubusercontent.com/99423097/164084652-a56df714-9178-4a9b-9d47-b280dc66f668.png)
![image](https://user-images.githubusercontent.com/99423097/164084727-f26eb00a-32ce-4076-8f48-43889c0674c5.png)

**Reply by willin** [reply]
but other repos can search from header search component. why?

**Comment by ghost** [comment]
Son of bitch I ll see in court

**Comment by ghost** [comment]
I have all screen shot u guys control my account without my permission let
meet in court I have emails
